plugins/javaembed/javaembed.cpp

```diff
@@ -3380,19 +3130,54 @@ class JavaEmbedImportContext : public CInterfaceOf<IEmbedFunctionContext>
     }
     virtual void importFunction(size32_t lenChars, const char *utf)
     {
-        sharedCtx->importFunction(lenChars, utf, classpath, 0, nullptr, instance);
-        argsig = sharedCtx->querySignature();
+        if (argsig)
```

Comment:
Worth commenting that this object will only be reused if the function name is a constant

Reply:
ok